src/error.rs

```diff
@@ -1,20 +1,26 @@
 /// Error variants produced by the Cocoon API.
 #[derive(Debug)]
+#[cfg_attr(feature = "std", derive(thiserror::Error))]
 pub enum Error {
     /// I/o error during read/write operation (`Cocoon::dump`, `Cocoon::parse`).
     #[cfg(feature = "std")]
+    #[error("Input/output error")]
     Io(std::io::Error),
     /// Format is not recognized. Probably corrupted.
+    #[cfg_attr(feature = "std", error("Unrecognized format"))]
     UnrecognizedFormat,
     /// Cryptographic error. There could be a few reasons:
     /// 1. Integrity is compromised.
     /// 2. Password is invalid.
+    #[cfg_attr(feature = "std", error("Cryptographic error: bad integrity/password?"))]
```

Comment:
Let's just not use "?"
```suggestion
    #[cfg_attr(feature = "std", error("Cryptographic error: bad integrity/password"))]
```